packages/terra-dialog-modal/tests/jest/DialogModal.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import DialogModalExample from './DialogModalExample';
+
```

Comment:
We should add jest tests for `isFocused`. Also, these tests are supplying all props at once is there any minimal combination we should test to verify this works as expected? It seems most props would always be supplied

Reply:
Except for `width`, `requestFocus`, `releaseFocus`, `isFocused` (and children), I think all other props are required.